ldap/servers/slapd/back-ldbm/db-mdb/mdb_import.c

```diff
@@ -736,7 +736,11 @@ dbmdb_import_all_done(ImportJob *job, int ret)
             ldbm_set_last_usn(inst->inst_be);
 
             /* bring backend online again */
-            slapi_mtn_be_enable(inst->inst_be);
+            if (job->flags & FLAG_REINDEXING) {
```

Comment:
Looking at bdb_import_all_done we have similar lines (enable backend even in case of reindex). Should it be also fixed in bdb_import_all_done ?

Reply:
No: Unlike in lmdb, bdb_import_all_done is not called after bdb_db2index (And that is why there is no problem with bdb)
That is due to a difference in the way xxx_db2index are coded:
 - bdb: The function is quite large (864 lines) because it prepares the reindex - performs the reindex - sets back backend in non_busy mode. BTW that is why bdb reindex is slower than import
 - mdb:  The function is simpler (85 lines) because it simply prepare the reindex then reuse the import framework pipeline (in lmdb, the reindex processing is quite similar to the import except than data are read from entryid instead of ldif)